src/ocm/model.js

```diff
@@ -240,31 +245,43 @@ const knownMetadataTypes = [
     name: 'structure_info',
     friendlyName: 'Structure Info',
     SpecificTypeHandler: ObjectTextViewer,
+    Icon: ArticleIcon
```

Comment:
should we rather define a fallback value that is always used if there is no specific icon?

Reply:
semantically, a known-metadata-type must define all of its attributes
for the fallback case we use `findTypedefByName` and `defaultTypedefForName`

Reply:
If you prefer to change these semantics I propse to address this in a separate PR because this affects quite some parts of our codebase